gtwrap/pybind_wrapper.py

```diff
@@ -193,6 +193,18 @@ def wrap_properties(self, properties, cpp_class, prefix='\n' + ' ' * 8):
                     ))
         return res
 
+    def wrap_operators(self, operators, prefix='\n' + ' ' * 8):
+        """Wrap all the overloaded operators in the `cpp_class`."""
+        res = ""
+        template = "{prefix}.def({{0}})".format(prefix=prefix)
+        for op in operators:
+            if op.operator in ('+', '-') and op.is_unary:
```

Comment:
so `is_unary` is not sufficient here?

Reply:
Good catch, it should be. Let me check and update.